plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergPageSource.java

```diff
@@ -236,7 +236,7 @@ public void deleteRows(Block rowIds)
             positionDeleteSink = positionDeleteSinkSupplier.get();
             verify(positionDeleteSink != null);
         }
-        positionDeleteSink.appendPage(new Page(rowIds));
+        positionDeleteSink.appendPage(new Page(rowIds)).join();
```

Comment:
> Fix Error Prone warnings in Iceberg

looks like some warning that we would want to have on error level perhaps

Reply:
Filed https://github.com/trinodb/trino/issues/13266